packages/calcite-components/src/components/date-picker/date-picker.tsx

```diff
@@ -454,12 +454,12 @@ export class DatePicker extends LitElement {
     return (Array.isArray(this.valueAsDate) && this.valueAsDate[1]) || undefined;
   }
 
-  private setEndDate(date: Date): void {
+  private setEndDate(date: Date, emitChangeEvent = true): void {
```

Comment:
Nitpick: "event" is implied by "emit", so this could be simplified to `emitChange`. Applies to `setStartDate`.